src/phoenix/server/api/types/Project.py

```diff
@@ -45,7 +45,7 @@ async def start_time(
         stmt = (
             select(func.min(models.Trace.start_time))
             .join(models.Project)
```

Comment:
I think we can do this query without the join now.

Reply:
you're right

Reply:
updated. we may need to put an index on the foreign keys if the index is not created by default